src/AvField.js

```diff
@@ -89,14 +89,14 @@ export default class AvField extends Component {
       size={size}
       disabled={disabled}
       readOnly={readOnly}
-      {...attributes}
+      {...{...attributes, ...{'aria-describedby':'errorMsg'}}}
```

Comment:
I think we should change this to 
```
aria-describedby="errorMsg"
{ ...attributes }
```

This way the user can override it if needed

Reply:
Yeah but in that case the aria attribute will not be read in AvInput as we are not expecting it under props.